packages/asset-swapper/test/sorting_utils_test.ts

```diff
@@ -0,0 +1,134 @@
+import { BigNumber } from '@0x/utils';
+import * as chai from 'chai';
+import 'mocha';
+
+import { sortingUtils } from '../src/utils/sorting_utils';
+
+import { chaiSetup } from './utils/chai_setup';
+import { testOrderFactory } from './utils/test_order_factory';
+
+chaiSetup.configure();
+const expect = chai.expect;
+
+const FAKE_ERC20_TAKER_ASSET_DATA = '0xf47261b22222222222222222222222222222222222222222222222222222222222222222';
+const FAKE_ERC20_MAKER_ASSET_DATA = '0xf47261b11111111111111111111111111111111111111111111111111111111111111111';
+
+describe('sortingUtils', () => {
```

Comment:
Nice work! Very cleanly communicates / tests the sorting.

Reply:
I didn't write like 99% of this PR (it just looks that way because of the rebase), so props to @dave4506